tests/unit/utils/test_pycrypto.py

```diff
@@ -22,10 +22,7 @@ class PycryptoTestCase(TestCase):
     TestCase for salt.utils.pycrypto module
     """
 
-    @skipIf(
-        not salt.utils.pycrypto.HAS_CRYPT or not salt.utils.pycrypto.HAS_PASSLIB,
-        "crypt not available",
-    )
+    @skipIf(not salt.utils.pycrypto.HAS_CRYPT, "crypt not available")
```

Comment:
keeping `or not HAS_PASSLIB` allows these tests to run if passlib is installed and the platform is windows.

I suggest we keep the condition as is, install passlib by default for tests, and modify the `force` below (line 56) to only be active if HAS_CRYPT is False (so the HAS_PASSLIB code is exercised on windows).

Reply:
(Right now the passlib code is *only* tested if passlib is manually installed, and these test changes break passlib).

Reply:
Hm... Skip if not crypt or not passlib makes the test running only if both installed.
Did you mean `@skipIf(not HAS_CRYPT and not HAS_PASSLIB)` to run it ever if CRYPT is not found but we have passlib?